docs/release-notes/release-notes-0.7.0.md

```diff
@@ -195,6 +195,11 @@
 
 - The `AddrReceives` RPC now supports timestamp filtering with
   [new `StartTimestamp` and `EndTimestamp` fields](https://github.com/lightninglabs/taproot-assets/pull/1794).
+  
+- The `BurnAsset` RPC now supports a 
+  [new `AssetSpecifier` field](https://github.com/lightninglabs/taproot-assets/pull/1812)
+  that allows the user to  specify the asset to burn by ID or GroupKey.
```

Comment:
Should explicitly mention that burning specific asset IDs of grouped assets is no longer supported.

Also should move release notes to `docs/release-notes/release-notes-0.7.1.md`

Reply:
Moved to `0-8-0.md `. Normally burning by assetID should work the same.